src/lib/__mocks__/mockApiRequests.ts

```diff
@@ -315,6 +327,8 @@ export async function mockFetch(
     switch (endpointParams.externalInstitutionId) {
       case 'error':
         return genericError;
+      case 'externalId1':

```

Comment:
more than with externalInstituionId, let's configure the error to return 409 if a user having for example CF XXXXXX00A00X000X is in the requestBody and the name/surname is not CERTIFIED_NAME and CERTIFIED_SURNAME, in this way, we could test that if we use the right values, we could end with success (this configuration is not meaningfull, but it's closer to the real use case :) )